packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -40,16 +37,16 @@ export async function makeWallet(
   }
 
   async function updatePursesState(pursePetname, purse) {
-    const [{ extent }, assay] = await Promise.all([
-      E(purse).getBalance(),
-      E(purse).getAssay(),
+    const [{ extent }, brand] = await Promise.all([
+      E(purse).getCurrentAmount(),
+      E(purse).getAllegedBrand(),
     ]);
-    const assayPetname = assayToAssayPetname.get(assay);
-    const assayRegKey = assayPetnameToRegKey.get(assayPetname);
+    const issuerPetname = issuerToIssuerPetname.get(brandToIssuer.get(brand));
+    const issuerRegKey = issuerPetnameToRegKey.get(issuerPetname);
     pursesState.set(pursePetname, {
       purseName: pursePetname,
-      assayId: assayRegKey,
-      allegedName: assayPetname,
+      issuerId: issuerRegKey,
+      issuerPetname,
```

Comment:
I think not having an 'allegedName' breaks the front-end unless we change that as well.